source/tutorials/first-steps/nix-language.md

```diff
@@ -2046,11 +2046,13 @@ This example is a (simplified) package declaration from Nixpkgs.
 
 Explanation:
 
-- This expression is a function that takes an attribute set which must have exactly the attributes `lib` and `stdenv`.
+- This expression is a function that takes an attribute set which must have exactly the attributes `lib`, `stdenv`, and `fetchurl`.
 - It returns the result of evaluating the function `mkDerivation`, which is an attribute of `stdenv`, applied to a recursive set.
-- The recursive set passed to `mkDerivation` uses its own `pname` and `version` attributes in the argument to the built-in function `fetchTarball`.
+- The recursive set passed to `mkDerivation` uses its own `pname` and `version` attributes in the argument to the function `fetchurl`, which is an attribute of the argument attribute set.
 - The `meta` attribute is itself an attribute set, where the `license` attribute has the value that was assigned to the nested attribute `lib.licenses.gpl3Plus`.
 
+The package can be built by running `nix-build -E 'with import <nixpkgs> {}; callPackage ./hello.nix {}'`.
```

Comment:
While it's good to make sure the example is correct, I'm not sure we should have it here. It introduces an entirely new concept with `callPackage`, which is quite magical.

For a moment I contemplated if we could make a "naive" example with the tools provided here, but it would be misleading because no one does it that way. Also I wouldn't want to go deeply into assessing the various ways to access packages here. It's a can of worms on its own. This tutorial is only about the language.

I think it's better to leave it out. @NixOS/documentation-team what do you think?